packages/lib-user/src/components/Contributors/helpers/generateExport.js

```diff
@@ -0,0 +1,58 @@
+import { arrayOf, func, number, shape, string } from 'prop-types'
+
+import { getExportData } from './getExportData'
+
+const DEFAULT_HANDLER = () => true
+
+export function generateExport({
+  group,
+  handleFileName = DEFAULT_HANDLER,
+  handleDataExportUrl = DEFAULT_HANDLER,
+  projects,
+  stats,
+  users
+}) {
+  const data = getExportData({ projects, stats, users })
+
+  let str = ''
+
+  data.forEach((row) => {
+    str += row.map((col) => JSON.stringify(col)).join(',').concat('\n')
+  })
+
+  let newFilename = `${group.display_name}.data_export.${Date.now()}.csv`
```

Comment:
This needs to be updated with the `group.display_name` sanitized to remove emojis at minimum.

Reply:
Refactored with 782f3f5